.eslintrc

```diff
@@ -2,6 +2,7 @@
     "env": {
        "node": true,
        "es6": true,
+       "browser": true,
```

Comment:
I prefer to opt-in to browser features on a per-file basis - rather than allowing them in any file.